src/clj/game/cards/identities.clj

```diff
@@ -1278,7 +1278,7 @@
 (defcard "Pālanā Foods: Sustainable Growth"
   {:events [{:event :runner-draw
              :req (req (and (first-event? state :corp :runner-draw)
-                            (pos? target)))
+                            (pos? (:count target))))
```

Comment:
I'm blown that this didn't cause any crashes in the tests.

Reply:
We have zero tests for Pālanā which is why this wasn't caught. I suspect tests weren't made because the ability is pretty straightforward.